### DISCUSSION TITLE
맵을 가로로 어떻게 전환할까요?

### DISCUSSION THREAD

**Discussion by hipsc159** [naverz/zepeto-studio-kor#257] [5. Q&A]
맵을 만들고 빌드를 했는데, 제페토에서 세로 모드로 되네요. 가로 모드로 어떻게 바꿀 수 있을까요?

**Comment by hyeonjin-jo** [comment]
안녕하세요.

아래 링크의 설정하기 부분에 Orientation옵션에 관한 내용 참고 부탁드립니다.

https://studio.zepeto.me/guides/multiplay

현재는 빌드전에만 고정적으로 가로, 세로를 지정할 수 있습니다

**Reply by hipsc159** [reply]
네 감사합니다!!
